postprocessing/janus-pp-rec.c

```diff
@@ -679,6 +680,10 @@ int main(int argc, char *argv[])
 			/* Things are simpler for data, no reordering is needed: start by the data time */
 			gint64 when = 0;
 			bytes = fread(&when, sizeof(gint64), 1, file);
+			if (bytes < 1) {
```

Comment:
1) _code style_: do not insert the space between `if` and the parenthesis
2) please change to
```c
if(bytes < (int)sizeof(gint64))
```

Reply:
ouch ! done !